script/Arcueid/Day 10/10_00_ARC10_4.txt

```diff
@@ -199,18 +223,22 @@ Arcueid's facial expression grows darker and darker.
 [sha:58b71adc544e3d3bdfae244d9408713072ba453e]{
 -- Page 310, Offset 25042. Glued. Mods: @x.
 -- アルクェイドにつられてか、俺も気持ちが沈んでしまった。うまく言えないが、こいつのそういう顔を見るのはイヤだった。
-Perhaps it's because I'm drawn to Arcueid, but even I'm starting to feel depressed. I can't put it into words, but seeing her face like that makes me feel bad.
+I must've got caught up in the moment with Arcueid. I'm starting to feel depressed as well.
+ I don't feel like I could eloquently express %{i}why%{/i}, but I just can't bear seeing her face like this.
 }
 [sha:6187b5f15e1df274a5699c35836434597a58c0bf]{
 -- Page 311, Offset 25043.
 -- 「早計じゃないか？　たんにさ、理論だけで経験を補えるヤツと、経験で理論を覆せるヤツがいるだけだと思うけど？
-"Isn't that conclusion bit premature, though? I think it's simply that there are those who make up for lack of experience with theory alone, and those who overturn theory with their experience.
+"Isn't that conclusion a bit premature, though?
+ I think what it all boils down to is that there are some people who make up for a lack of experience with theory alone,
```

Comment:
I think that what it all boils down to is that there are people who can make up for their lack of experience with theory alone, and those who can overturn theory through the benefit of their experiences.